modules/raycast/SCsub

```diff
@@ -87,9 +84,16 @@ if env["builtin_embree"]:
     env_thirdparty.disable_warnings()
     env_thirdparty.add_source_files(thirdparty_obj, thirdparty_sources)
 
+    # Set x86 CPU instruction sets to use when building Embree's own intrinsics.
+    # Keep this in sync with Godot's main SConstruct file.
     if env["arch"] != "x86_64" or env.msvc:
-        # Embree needs those, it will automatically use SSE2NEON in ARM
-        env_thirdparty.Append(CPPDEFINES=["__SSE2__", "__SSE__"])
+        # Embree needs those; it will automatically use SSE2NEON in ARM.
+        env_thirdparty.Append(CPPDEFINES=["__SSE__", "__SSE2__"])
+
+        if env["arch"] == "x86_64":
```

Comment:
This check should probably be unindented one level, currently it's inside `env["arch"] != "x86_64"` (line 89) so will be never true. Or is it only for MSVC?.